spec/lib/terraforming/resource/iam_group_spec.rb

```diff
@@ -47,36 +59,38 @@ module Resource
         end
       end
 
-      describe ".tfstate" do
-        it "should generate tfstate" do
-          expect(described_class.tfstate(client: client)).to eq({
-            "aws_iam_group.hoge" => {
-              "type" => "aws_iam_group",
-              "primary" => {
-                "id" => "hoge",
-                "attributes" => {
-                  "arn"=> "arn:aws:iam::123456789012:group/hoge",
-                  "id" => "hoge",
-                  "name" => "hoge",
-                  "path" => "/",
-                  "unique_id" => "ABCDEFGHIJKLMN1234567",
+      describe '.tfstate' do
+        it 'should generate tfstate' do
+          expect(described_class.tfstate(client: client)).to eq(
+            {
+              'aws_iam_group.hoge' => {
+                'type' => 'aws_iam_group',
+                'primary' => {
+                  'id' => 'hoge',
+                  'attributes' => {
```

Comment:
__[rubocop]__
* Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.